### DISCUSSION TITLE
Find and Replace in Genre Tag

### DISCUSSION THREAD

**Discussion by sly-fox-git** [beetbox/beets#5138] [Q&A] [answered]
Can you please tell me what would be a proper command to find "," in the genre tag in my entire library and replace it with "/"

ChatGPT suggested `beet modify genre::',.*' genre+=\$/ -W` I am not sure if this is correct.

**Comment by JOJ0** [comment, marked as answer]
Isn't ChatGPT always right? haha ;-)

In that suggested command, what I know is that the `beet mod` command is correct syntax more or less, but not sure about the regex... Why don't you test the regex on just a couple of albums. Modify _alsways_ asks for confirmation before it actually does the modifications (unless you omit that question with the `-y` option.


This is how `modify` or short `mod` command basically works

```
beet modify genre::query_in_form_of_regex genre=new_value
```

I personally always switch the order of the options, I think it adds to readability since the last option always is my query, which often could be rather longish:


```
beet mod genre=new_value genre::regex_query more_query_args
```

I am not sure about the `+=` in your suggested command....I've never seen it, but that doesn't mean it's not valid. Plz anyone else, doublecheck if that is valid syntax.

Relevant docs:

https://beets.readthedocs.io/en/latest/reference/cli.html#modify

https://beets.readthedocs.io/en/latest/reference/query.html

Hope that helps you move on with the topic :-)
